docker-compose.yml

```diff
@@ -226,7 +226,7 @@ services:
           - "${RABBITMQ_PORT-5674}:5672"
 
   koku-worker:
-      container_name: koku-worker
+      # container_name: koku-worker
```

Comment:
You've got this commented out. Is this intentional to be left in or is this a hanger-on during development?